pandas/tests/io/formats/style/test_style.py

```diff
@@ -1597,7 +1597,7 @@ def test_no_empty_apply(mi_styler):
 
 
 @pytest.mark.parametrize("format", ["html", "latex", "string"])
-def test_output_buffer(mi_styler, format):
+def test_output_buffer(mi_styler, format, temp_file):
     # gh 47053
-    with tm.ensure_clean(f"delete_me.{format}") as f:
-        getattr(mi_styler, f"to_{format}")(f)
+    f = str(temp_file)
```

Comment:
Generally, if the test isn't specifically testing a string file path, could you just using `temp_file` directly instead of doing `str(temp_file)`?